src/Notices.php

```diff
@@ -87,6 +87,7 @@ public function get( $id ) {
 		if ( isset( $this->notices[ $id ] ) ) {
 			return $this->notices[ $id ];
 		}
+		return null;
```

Comment:
Why was this line added?
Functions without a return statement will return `null` at runtime anyway :thinking:

Reply:
@aristath Implicit programming **vs.** Explicit programming -> explicit wins! Humans can read it and it's testable.